galsim/phase_psf.py

```diff
@@ -1338,8 +1338,9 @@ def __init__(self, lam_over_diam=None, lam=None, diam=None, tip=0., tilt=0., def
         if max_size is not None:
             from .deprecated import depr
             depr('max_size', 1.4, '',
-                 "The max_size keyword has been removed.  If you have a need for it, "
-                 "please open an issue requesting the functionality.")
+                 "The max_size keyword has been removed.  In its place, the pad_factor keyword can"
+                 "be used to adjust the size of the internal InterpolatedImage.")
```

Comment:
This looks good to me.  But some of the examples in des and great3 directories currently use max_size.  These should be updated to remove that. Maybe with a comment that the original great3 runs used max_size, which we now think was invalid, leading to some aliasing problems that went unnoticed.